client/checkout/platform-checkout/email-input-iframe.js

```diff
@@ -204,6 +208,19 @@ export const handlePlatformCheckoutEmailInput = ( field, api ) => {
 		}
 	} );
 
+	const platformCheckoutCheckSession = () => {
+		return fetch(
+			`${ getConfig(
+				'platformCheckoutHost'
+			) }/wp-json/platform-checkout/v1/user/session`,
```

Comment:
Does this needs to be done on the front-end? Or can this be done via a hook of some sort on the PHP side? 

I am thinking something like, as soon as `/checkout` is loaded, PHP sends the request to `/wp-json/platform-checkout/v1/user/session` before we render anything. This way, we don't need to disable the screen and the checkout redirection can be more fluid.